Makefile

```diff
@@ -95,7 +95,7 @@ clean:
 
 lint: .docker-build-pull
 	${DC} run test flake8
-	${DC} run assets gulp js:lint css:lint json:lint
+	${DC} run assets npm run lint
 
 test: .docker-build-pull
 	${DC} run test
```

Comment:
We could also add `${DC} run assets npm run test` here

Reply:
I don’t think we can do that as the tests require browsers, which aren’t in the container?

Reply:
AH! I forgot about that. We can figure that out later.